src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import { FilterBy } from '../../types/FilterBy';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  filterBy: FilterBy;
+  setFilterBy: (arg: FilterBy) => void;
+  deleteCompletedTodos: () => void;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos = [],
+  filterBy,
+  setFilterBy,
+  deleteCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {todos.filter(todo => !todo.completed).length} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: filterBy === 'all',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilterBy('all')}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filterBy === 'active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={() => setFilterBy('active')}
+        >
+          Active
+        </a>
+
+        <a
+          href="#/completed"
+          className={classNames('filter__link', {
+            selected: filterBy === 'completed',
+          })}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setFilterBy('completed')}
+        >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method